src/rard/templates/research/partials/ordered_work_area.html

```diff
@@ -31,10 +33,12 @@
                     <div>
                         {% if work.unknown == True and not flinks and not tlinks and not alinks %}
                         {% elif work.unknown == True %}
-                        <b class='my-3 mr-3'><a href='{% url "work:detail" work.pk %}'>{{ work.name }}</a></b>
+                        <b class='my-3 mr-1'><a href='{% url "work:detail" work.pk %}' >{{ work.name }}</a></b>
+                        <button type='button' class="btn btn-text text-primary pl-2 collapse-button" data-toggle="collapse" data-target="#list_{{work.pk}}" role="button" aria-expanded="false" aria-controls="list_{{work.pk}}"><i class='fas fa-chevron-down'><small class='ml-1 text-muted'>{% trans "Expand" %}</small></i></button>
```

Comment:
Does `btn-text` exist in Bootstrap4? How about `btn-light`? And possibly with `btn-sm` to make it the same size as the other buttons in that row when in edit mode?

Reply:
I think `btn-text` does exist since it hasn't done the outline or filled it in....but looking at the docs it seems actually to be `btn-link` 🤔 I'll update it to be doubly sure